region/crateria/west.json

```diff
@@ -799,24 +959,40 @@
           "to": [
             {
               "id": 2,
-              "requires": [
-                {"or": [
-                  "ScrewAttack",
-                  "canUseMorphBombs",
-                  {"and": [
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": ["ScrewAttack"]
+                },
+                {
+                  "name": "Morph Bomb Gauntlet Entrance (In)",
```

Comment:
Going left to right is not what I'd consider "In"

Reply:
Oops